features-linux.md

```diff
@@ -0,0 +1,183 @@
+# <a name="linuxFeatures" />Linux Features Document
```

Comment:
Is this missing feature flag for `IntelRDT` , some runtimes supports restricting bandwidth using that. Also `runtime-spec` has a definition for it.

Not sure maybe something like

```json
"intelrdt": {
  "enabled": true
}
```

Reply:
Added `.linux.intelRdt.enabled` field